.bit/responses/1.9-Week 1 Step 9.md

```diff
@@ -15,8 +15,12 @@ name: Week 1 Step 9
 
 > *Welcome agent!* You have made it this far so we know we can trust you. BitProject is working in an undercover operation, and we need a new way to communicate.
 
-**Help us by creating an Azure Function that can help us convert regular English into Morse Code**. 
+### ✅  Tasks:
 
+- [ ]  Code an HTTP trigger function with `morse-code-converter` that takes in English as a parameter and outputs it in Morse Code.
+- [ ]  To move on, place the function URL of your new Morse Code HTTP Trigger as a repository secret named `MORSE_URL`. Then, commit your function's code to a file named `week1/morse.js` to the `week1` branch.**
```

Comment:
Go through the *tasks* in all the files and make sure these tasks are separated (the second check box should be split into three separate tasks)
- make sure you're on the week1 branch
- commit the code to...
- place repo secret